Doesn't compress using YUI-compressor

It seems that this library doesn't compress when using the YUI compressor. I get an exception when I try this:

    $ yui-compressor --type=js libs/json5/lib/json5.js

It's a Java exception. I'm not sure what the problem is. The same command works fine on every other javascript file I've thrown at it.

```
$ yui-compressor --type=js lib/json5.js --verbose

[INFO] Using charset UTF-8
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at com.yahoo.platform.yui.compressor.Bootstrap.main(Bootstrap.java:21)
Caused by: java.util.MissingResourceException: Can't find bundle for base name org.mozilla.javascript.resources.Messages, locale en_US
	at java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:1499)
	at java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1322)
	at java.util.ResourceBundle.getBundle(ResourceBundle.java:795)
	at org.mozilla.javascript.ScriptRuntime$DefaultMessageProvider.getMessage(ScriptRuntime.java:3608)
	at org.mozilla.javascript.ScriptRuntime.getMessage(ScriptRuntime.java:3592)
	at org.mozilla.javascript.ScriptRuntime.getMessage0(ScriptRuntime.java:3540)
	at org.mozilla.javascript.Parser.addError(Parser.java:145)
	at org.mozilla.javascript.Parser.reportError(Parser.java:160)
	at org.mozilla.javascript.Parser.mustMatchToken(Parser.java:265)
	at org.mozilla.javascript.Parser.function(Parser.java:598)
	at org.mozilla.javascript.Parser.parseFunctionBody(Parser.java:479)
	at org.mozilla.javascript.Parser.function(Parser.java:611)
	at org.mozilla.javascript.Parser.primaryExpr(Parser.java:2255)
	at org.mozilla.javascript.Parser.memberExpr(Parser.java:1974)
	at org.mozilla.javascript.Parser.unaryExpr(Parser.java:1832)
	at org.mozilla.javascript.Parser.mulExpr(Parser.java:1761)
	at org.mozilla.javascript.Parser.addExpr(Parser.java:1742)
	at org.mozilla.javascript.Parser.shiftExpr(Parser.java:1722)
	at org.mozilla.javascript.Parser.relExpr(Parser.java:1696)
	at org.mozilla.javascript.Parser.eqExpr(Parser.java:1652)
	at org.mozilla.javascript.Parser.bitAndExpr(Parser.java:1641)
	at org.mozilla.javascript.Parser.bitXorExpr(Parser.java:1630)
	at org.mozilla.javascript.Parser.bitOrExpr(Parser.java:1619)
	at org.mozilla.javascript.Parser.andExpr(Parser.java:1607)
	at org.mozilla.javascript.Parser.orExpr(Parser.java:1595)
	at org.mozilla.javascript.Parser.condExpr(Parser.java:1578)
	at org.mozilla.javascript.Parser.assignExpr(Parser.java:1563)
	at org.mozilla.javascript.Parser.assignExpr(Parser.java:1569)
	at org.mozilla.javascript.Parser.expr(Parser.java:1542)
	at org.mozilla.javascript.Parser.statementHelper(Parser.java:1221)
	at org.mozilla.javascript.Parser.statement(Parser.java:726)
	at org.mozilla.javascript.Parser.parse(Parser.java:420)
	at org.mozilla.javascript.Parser.parse(Parser.java:378)
	at com.yahoo.platform.yui.compressor.JavaScriptCompressor.parse(JavaScriptCompressor.java:312)
	at com.yahoo.platform.yui.compressor.JavaScriptCompressor.<init>(JavaScriptCompressor.java:533)
	at com.yahoo.platform.yui.compressor.YUICompressor.main(YUICompressor.java:131)
	... 5 more
```

--- a/lib/json5.js
+++ b/lib/json5.js
@@ -547,17 +547,17 @@ JSON5.stringify = function (obj, replacer, space) {
         }
     };
 
-    function isWordChar(char) {
-        return (char >= 'a' && char <= 'z') ||
-            (char >= 'A' && char <= 'Z') ||
-            (char >= '0' && char <= '9') ||
-            char === '_' || char === '$';
+    function isWordChar(c) {
+        return (c >= 'a' && c <= 'z') ||
+            (c >= 'A' && c <= 'Z') ||
+            (c >= '0' && c <= '9') ||
+            c === '_' || c === '$';
     }
 
-    function isWordStart(char) {
-        return (char >= 'a' && char <= 'z') ||
-            (char >= 'A' && char <= 'Z') ||
-            char === '_' || char === '$';
+    function isWordStart(c) {
+        return (c >= 'a' && c <= 'z') ||
+            (c >= 'A' && c <= 'Z') ||
+            c === '_' || c === '$';
     }
 
     function isWord(key) {

